src/components/Heroes/RankColumns.ts

```diff
@@ -0,0 +1,271 @@
+import heroes from 'dotaconstants/build/heroes.json';
+import constants from 'src/components/constants'
+import { abbreviateNumber, displayHeroId } from '../../utility';
+import { GlobalStrings } from 'src/types/common/GlobalStrings'
+import { Heroes } from 'src/types/Hero/Heroes'
+
+type Props = {
+  tabType: HeroesTabType
+  strings: GlobalStrings
+}
+
+enum HeroesTabType {
```

Comment:
can we avoid adding `Type` suffix here?

Reply:
Yes, I was struggling for a name for this. Suffixing with type is a bad habit I've gotten into for tabs and columns